client/app/bundles/course/assessment/submission/components/CommentCard.jsx

```diff
@@ -105,13 +106,13 @@ export default class CommentCard extends Component {
 
   renderCommentContent() {
     const { editMode } = this.state;
-    const { editValue, post: { formattedText, id } } = this.props;
+    const { editValue, disableAirMode, post: { formattedText, id } } = this.props;
 
     if (editMode) {
       return (
         <>
           <MaterialSummernote
-            airMode
+            airMode={disableAirMode !== true}
```

Comment:
Can write `airMode={!disableAirMode}`

But this double negative can be confusing to read. I would suggest to invert `disableAirMode` everywhere and use `airMode` prop instead, and set it as `true` using defaultProps.